runtime/libffi/z/sysvz64.s

```diff
@@ -679,16 +614,17 @@ STFPCHKD246 DS 0H
 *determine how to pass the struct based on size
 *at this point we've weeded out all the float/double
 *pair structs that get passed purely in FPRs
-STRCT2  DS 0H
-         LG 15,0(,11)          type->size
-         CFI 15,8              Struct <= 8 bytes?
-         BNH  BYTE8
-         CFI 15,16             Struct <= 16 bytes?
-         BNH   BYTE16
-         CFI 15,24             Struct <= 24 bytes?
-         B BYTE24
-
-BYTE8    DS 0H               Struct <= 8 bytes
+STRCT   DS 0H
+        LG  11,0(10,5)
+        LG  15,0(,11)         type->size
+        CFI 15,8              Struct <= 8 bytes?
+        BNH  BYTE8
+        CFI 15,16             Struct <= 16 bytes?
+        BNH   BYTE16
+        CFI 15,24             Struct <= 24 bytes?
+        B BYTE24
+
+BYTE8   DS 0H                 Struct <= 8 bytes
```

Comment:
The opcode and comments should align with existing code (e.g. lines 602-605).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/583867825be4477a8319f21ffb952780631fbeb3..f6eb07952b7a7b7c8c8789fbb19e2c66f7672683

Reply:
Not done: to match surrounding code, there should be one more space before each comment:
```
         LG  15,0(,11)          type->size
         CFI 15,8               Struct <= 8 bytes?
         BNH  BYTE8
         CFI 15,16              Struct <= 16 bytes?
         BNH   BYTE16
         CFI 15,24              Struct <= 24 bytes?
         B BYTE24

BYTE8    DS 0H                  Struct <= 8 bytes
```

Reply:
Changed in https://github.com/eclipse-openj9/openj9/compare/f6eb07952b7a7b7c8c8789fbb19e2c66f7672683..1a045fe4195b193257edc874d61fd73030c8124a and 
https://github.com/eclipse-openj9/openj9/compare/1a045fe4195b193257edc874d61fd73030c8124a..91cb2ada99ecad623c87439a68e9c15758d5f34a